Libraries/Blob/RCTBlobManager.m

```diff
@@ -0,0 +1,154 @@
+/**
+ * Copyright (c) 2015-2016 Silk Labs, Inc.
+ * All Rights Reserved.
+ */
+
+#import "RCTBlobManager.h"
+
+#import <React/RCTConvert.h>
+
+static NSString *const kBlobUriScheme = @"blob";
+
+@implementation RCTBlobManager
+{
+  NSMutableDictionary<NSString *, NSData *> *_blobs;
+  NSOperationQueue *_queue;
+}
+
+RCT_EXPORT_MODULE(BlobModule)
+
+- (NSDictionary<NSString *, id> *)constantsToExport
```

Comment:
Export constants causes modules to be initialized / allocated eagerly. Since these are just constants, perhaps you could just have these strings in JS?